composer.json

```diff
@@ -14,8 +14,8 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4@dev",
-        "silverstripe/admin": "^1@dev"
+        "silverstripe/framework": "^4",
+        "silverstripe/admin": "^1"
```

Comment:
Dan’s request was to make the constraint specific here too, so `^1.0.2` would be good!

Reply:
Yep, realised it as I committed 😅